runtime/errors.go

```diff
@@ -66,6 +66,9 @@ var (
 
 type errorBody struct {
 	Error   string     `protobuf:"bytes,1,name=error" json:"error"`
+	// This is to make the error more compatible with users that expect errors to be Status objects. It should be
```

Comment:
Could you add a link to https://github.com/grpc/grpc/blob/master/src/proto/grpc/status/status.proto?

Reply:
done!